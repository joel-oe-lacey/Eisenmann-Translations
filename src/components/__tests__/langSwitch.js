import React from "react"
import renderer from "react-test-renderer"
import LangSwitch from "../langSwitch"

describe("Language Switch", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LangSwitch />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})