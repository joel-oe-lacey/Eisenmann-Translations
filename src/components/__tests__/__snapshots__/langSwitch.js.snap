// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Language Switch renders correctly 1`] = `<section />`;
